swift/ql/test/library-tests/dataflow/taint/TaintInline.ql

```diff
@@ -0,0 +1,20 @@
+import swift
+import Taint
+import TestUtilities.InlineExpectationsTest
+
+class TaintTest extends InlineExpectationsTest {
+  TaintTest() { this = "TaintTest" }
+
+  override string getARelevantTag() { result = "taintedFromLine" }
+
+  override predicate hasActualResult(Location location, string element, string tag, string value) {
+    exists(TestConfiguration config, Node source, Node sink, Expr sinkExpr |
+      config.hasFlow(source, sink) and
+      sinkExpr = sink.asExpr() and
+      location = sinkExpr.getLocation() and
+      element = sinkExpr.toString() and
+      tag = "taintedFromLine" and
```

Comment:
`taintedFromLine` is quite a mouthful. Could we just call it `tainted` instead? If you think it's too unclear what `tainted=42` is supposed to mean we could write a comment at the top of the file saying that `tainted=42` means that taint is coming from line 42.

Reply:
I lean towards being slightly more verbose, but I'm happy with just `tainted` if you are.  Done.